### DISCUSSION TITLE
Advantages and disadvantages of the program Code

### DISCUSSION THREAD

**Discussion by pofart** [elementary/code#917] [General]
Hello respected community and developers. First, I want to express my gratitude for your work. I am interested in the program for text and code editing - "Code". 
I would like to ask about its differences from others, what are its advantages and disadvantages? 
Where can I read documentation about the program? Is there telemetry in the program? 
Would be a good choice if I want to completely switch to writing code and scripts in it? 
Will there be any problems when installing this program in other distributions? 
Thank you very much!

**Comment by cassidyjames** [comment]
>I would like to ask about its differences from others, what are its advantages and disadvantages?

That's super broad, but the main advantages are that it's designed and developed by the same people designing and developing the rest of elementary OS, so it is consistent with the other apps and perhaps easier to understand—basically see the benefits listed at the top of the [Human Interface Guidelines](https://elementary.io/docs/human-interface-guidelines#human-interface-guidelines).

>Where can I read documentation about the program? 

There is the [Code Wiki](https://github.com/elementary/code/wiki) here on GitHub.

>Is there telemetry in the program?

No. You may want to see the [elementary Privacy page](https://elementary.io/privacy) and our [blog post about the topic](https://blog.elementary.io/privacy-and-elementary-os/).

>Would be a good choice if I want to completely switch to writing code and scripts in it?

We use Code to write elementary OS, so obviously we think it's a pretty good choice. 😄 

>Will there be any problems when installing this program in other distributions?

We develop elementary Code for elementary OS. However, it is a GTK app and should run on other distributions, we just don't explicitly develop or test for that. This may become more straightforward in the future with a Flatpak build of Code, though.

**Reply by pofart** [reply]
Thanks for detail answer! 
About the first question, I meant how suitable it is for a beginner programmer for small projects :)

**Reply by hanaral** [reply]
I use it for HTML and other projects, it's my main text editor
